documentation/User-Documentation/Notifications.md

```diff
@@ -151,104 +215,76 @@ Shows the notification using a title.
+| sound | Sound to play. If blank, no sound. Either <a href="https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/Sound.html" target="_blank"> vanilla Minecraft sounds</a> or the name of a sound from a resource pack. |
 | fadeIn | Ticks to fade the title in. Default 10 |
 | stay | Ticks to keep title on screen. Default 70 |
 | fadeOut | Ticks to fade the title out. Default 20 |
 
 
-## Custom Notifications
-Using the `notify` event a custom notification can be sent. It can make use of any category defined or can override by directly defining the NotifyIO configuration options. Please refer to the Events-List chapter for more details on this event.
+### Sound
+This IO just plays a sound. You should read the <a href="https://minecraft.gamepedia.com/Commands/playsound" target="_blank">wiki page</a> of the playsound command
+as Minecraft's sound system is kinda strange. 
 
-## Examples
+| Option | Description |
+|--------|-------------|
+| sound | Sound to play. If blank, no sound. Either vanilla Minecraft sounds (get them using /playsound autocompletion) or the name of a sound from a resource pack. |
+| soundcategory | The <a href="https://hub.spigotmc.org/javadocs/bukkit/org/bukkit/SoundCategory.html" target="_blank">category</a> in which the sound will be played. |
+| soundvolume | Minecraft's <a href="https://minecraft.gamepedia.com/Commands/playsound#Arguments" target="_blank">special sound volume</a>. Default: _1_ |
+| soundpitch | Pitch of the sound. Default: _1_ Min: _0_ Max: _2_ |
+| soundlocation | Default: The player's location. A location using the BetonQuest [ULF](../Reference/#unified-location-formating). Can include variables. |
+| soundplayeroffset | This option is special. See below.
 
-### Example 1 - Custom Notifications
-Assuming the following custom.yml file:
-```YAML
-notifications:
-    # Test Categories
-    test_suppress:
-        io: suppress
-
-    test_chat:
-        io: chat
-        sound: BLOCK_CHEST_CLOSE
-
-    test_advancement:
-        io: advancement
-        sound: BLOCK_CHEST_CLOSE
-        frame: challenge # challenge|goal|task
-        icon: map
-
-    test_actionbar:
-        io: actionbar
-        sound: BLOCK_CHEST_CLOSE
-
-    test_bossbar:
-        io: bossbar
-        sound: BLOCK_CHEST_CLOSE
-        #barFlags: create_fog,darken_sky,play_boss_music
-        barColor: purple # blue|green|pink|purple|red|white
-        progress: 0.0 # 0.0 - 1.0
-        style: solid # segmented_10|segmented_12|segmented_20|segmented_6|solid
-        stay: 70
-
-    test_title:
-        io: title
-        sound: BLOCK_CHEST_CLOSE
-        fadeIn: 10
-        stay: 70
-        fadeOut: 20
-        #subtitle:
-
-    test_subtitle:
-        io: subtitle
-        sound: BLOCK_CHEST_CLOSE
-        fadeIn: 10
-        stay: 70
-        fadeOut: 20
-```
+<h3>soundplayeroffset:</h3>
+This option can be a number or a vector.
```

Comment:
This options are allowed for every IO

Reply:
use the sound category in every other IO's table to link to the sound IO